docs/node/run-your-node/prerequisites/system-configuration.mdx

```diff
@@ -3,94 +3,36 @@ import TabItem from '@theme/TabItem';
 
 # System Configuration
 
-This page describes changes that should be made to the configuration of the
-system where you are running an Oasis Node instance.
+This page outlines the modifications necessary that should be made to the
+configuration of the system where you are running an Oasis Node instance. To
+prepare the system for running an Oasis Node, we will guide you through the
+process of creating a dedicated user account and increasing the file descriptor
+limit.
 
-## File Descriptor Limit
+## Create an user
+Nothing in Oasis Node requires elevated privileges, so running the Oasis Node
+with root privileges is not allowed. Attempting to run the `oasis-node` process
+as the root user will terminate immediately on startup. We strongly advise
+establishing a dedicated user account to manage the Oasis Node processes. 
 
-Make sure that the user under which you are running your Oasis Node has a high-enough file descriptor limit as the databases can have many files open and running out of file descriptors can lead to the node stopping unexpectedly.
-
-You can check the file descriptor limit by running the following as the same user that will run Oasis Node:
+To create the user run as root:
 
 ```bash
```

Comment:
Let's use `shell` language in our docs, if bash syntax is not specifically required.

```suggestion
```shell
```